Makefile-az.mk

```diff
@@ -24,7 +24,7 @@ az-login: ## Login into Azure
 az-mkrg: ## Create resource group
 	az group create --name $(AZURE_RESOURCE_GROUP) --location $(AZURE_LOCATION) -o none
 
-az-mkacr: az-mkrg ## Create test ACR
+az-mkacr: ## Create test ACR
```

Comment:
This breaks `make az-mkaks`. However, I think that might not be used anywhere, so could possibly be removed?

Reply:
Or you could add `az-mkrg` as a command to the `az-mkaks` before `az-mkacr`?

Reply:
Does leaving this here cause an error, if we try creating the calling `az-mkacr` twice?

If not, maybe it should be left?

Reply:
As it would also now break `az-mkaks-cilium` actually

Reply:
Yeah I see that now. Running mk-rg twice will fail w/ rg already created error... can we run `az-mkacr` directly after `az-login`?

Reply:
I think this change worked bc I accidentally ran `make- az-mkrg` before this change. Let me edit this, my bad

Reply:
Well, `make az-all` will currently work since it will have the `az-create-workload-msi` call before, but its just the direct calls that will fail.